pkg/rpc/api.go

```diff
@@ -6,12 +6,17 @@ import (
 	"errors"
 	"fmt"
 
+	dbAbi "github.com/NethermindEth/juno/internal/db/abi"
 	"github.com/NethermindEth/juno/internal/services"
+	"github.com/NethermindEth/juno/pkg/feeder"
 	"github.com/NethermindEth/juno/pkg/types"
 
 	"github.com/NethermindEth/juno/internal/log"
 )
 
+// Global feederClient that we use to request pending blocks
+var feederClient = feeder.NewClient("https://alpha-mainnet.starknet.io", "/feeder_gateway", nil)
```

Comment:
You shouldn't have hard-coded the feederClient should be part of the config. initialized on the run of the rpc or something like that